apex/contrib/nccl_allocator/nccl_allocator.py

```diff
@@ -8,9 +8,13 @@
 __all__ = ["init", "nccl_mem", "create_nccl_mem_pool"]
 
 
-def create_nccl_mem_pool():
+def create_nccl_mem_pool(symmetric=False):
     _allocator = _apex_nccl_allocator.get_nccl_allocator()
-    _pool = torch.cuda.MemPool(_allocator)
+    _pool = (
+        torch.cuda.MemPool(_allocator, symmetric=True) 
+        if symmetric 
+        else torch.cuda.MemPool(_allocator)
+    )
```

Comment:
isn't this equivalent and shorter?
```suggestion
    _pool = torch.cuda.MemPool(_allocator, symmetric=symmetric)
```

Reply:
Okay I see, then I think a meticulous thing we could do is to have a signature check whether or not `symmetric` argument is available and then show a warning if `symmetric` is set after making creat_nccl_mem_pool's symmetric argument's default value to `None`.

Reply:
Hi @crcrpar, I have updated the PR with your suggestion. Thanks!